ext/opcache/ZendAccelerator.c

```diff
@@ -2425,6 +2486,152 @@ int accel_post_deactivate(void)
 		return SUCCESS;
 	}
 
+	/* Free static variables of preloaded classes and functions */
+	if (ZCSG(preload_script)) {
```

Comment:
The cleanup that is done here for immutable functions/classes, can all of it be skipped for fast shutdown? I.e. just zeroing the request data should be sufficient?

Reply:
You must be right, but current implementation also may preload non-immutable classes, and we have to deal with them. I'll think how to solve this.